services/QuillLMS/app/models/question.rb

```diff
@@ -149,4 +154,37 @@ def rematch_type
   private def stored_as_array(key)
     data[key].class == Array
   end
+
+  private def validate_sequences
+    return if !data.present? || !data.is_a?(Hash)
```

Comment:
I would use `data.blank?` over `!data.present?`  It's easier to reason through conditionals in the positive than negated ones.